mne/utils/docs.py

```diff
@@ -2486,7 +2503,7 @@
     Name of the output file.
 """
 docdict['export_params_fmt'] = """
-fmt : 'auto' | 'eeglab' | 'edf'
+fmt : 'auto' | 'eeglab'
```

Comment:
Looks like a bad merge (probably from the `git checkout ...` line), you'll need to manually put the lines back on this one :(

Reply:
I cherry-picked the respective commit from `main` (see 0658f29) and I think this got all those changes now.